models/calendar.yml

```diff
@@ -21,6 +21,7 @@
     - 8:45 PM:
         type: Talk
         emoji: 🛠
+        title: 5 Things About Chrome DevTools That Will Surprise You
```

Comment:
no it's just a dumb placeholder for now. still waiting on the title and description